### DISCUSSION TITLE
Where to find organs weights and tissue weights?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#774] [Questions & Problems] [answered]
I want to find organ weights and tissue weights in PK-Sim, but I didn't find them. Could you tell me where I could find them?

**Comment by JanSchlender** [comment, marked as answer]
Hi Wenjing,
The parameters you are looking for are characteristics of individuals and are thus located in the respective building block. 
![grafik](https://user-images.githubusercontent.com/26056951/89390827-9901db00-d707-11ea-90c7-f05a4e791d1d.png)

**Comment by Wenjing1018** [comment]
@JanSchlender,
I know it's in the individual building block, but  the volume of organs and tissues is involved in this building block, but the weight of organs and tissues is not found. where I could find them?

**Comment by JanSchlender** [comment]
You are welcome. The density is 1 g/ml

**Comment by Wenjing1018** [comment]
@JanSchlender ,
I got it,thank you very much!
yours sincerely!
